EnglishTranslation-master/UI.tsv

```diff
@@ -2814,16 +2814,16 @@ UI_20170912_002813	{@st66}News	{@st66}공지
 UI_20170912_002814	{@st66}Notes	{@st66}소개글				
 UI_20170912_002815	News	공지글				
 UI_20170912_002816	Notes	소개글				
-UI_20170912_002817	Member {img %s 13 13}	길드원{img %s 13 13}				
+UI_20170912_002817	Name {img %s 13 13}	길드원{img %s 13 13}				
 UI_20170912_002818	Level {img %s 13 13}	레벨{img %s 13 13}				
-UI_20170912_002819	Position {img %s 13 13}	직위{img %s 13 13}				
+UI_20170912_002819	Memo {img %s 13 13}	직위{img %s 13 13}				
```

Comment:
this is not the translation, and position makes sense to allow the guild leader to specify the name of a "rank" or "position" within the guild hierarchy

Reply:
Right, when people mentioned you could write anything in it I immediately thought of a short notice/memo. The Korean does mention 'Position' or 'Rank'. I changed it to 'Rank' since it is commonly used in hierarchical relationships in guilds.

Reply:
My concern with rank over position is it overuses the word "Rank" when describing something about someone. This could easily be confused for "class rank" when looking at a list of people, as in rank 8, rank 9, etc. Hence, position makes the most sense.